robottelo/hosts.py

```diff
@@ -1599,6 +1599,21 @@ def get_features(self):
         """Get capsule features"""
         return requests.get(f'https://{self.hostname}:9090/features', verify=False).text
 
+    def enable_capsule_downstream_repos(self):
+        """Enabling CDN repos and capsule downstream repos on Capsule Host"""
+        # CDN Repos
+        self.register_to_cdn()
+        for repo in getattr(constants, f"OHSNAP_RHEL{self.os_version.major}_REPOS"):
+            result = self.enable_repo(repo, force=True)
+            if result.status:
+                raise CapsuleHostError(f'Repo enable at capsule host failed\n{result.stdout}')
```

Comment:
When you register to cdn, this repos are enabled as default so we won't need additional step to enable this rhel repos
```suggestion

```

Reply:
Agree and I observed the same! But there are existing functions in robottelo which are trying to do it explicitly hence I had to repeat here. 

If we want to remove this, we will remove from all occurrences of it in a separate PR.